tests/jlm/mlir/backend/TestJlmToMlirConverter.cpp

```diff
@@ -97,7 +385,12 @@ TestLambda()
 static int
 Test()
 {
+  std::cout << "*** Running TestLambda() ***" << std::endl;
   TestLambda();
+  std::cout << "*** Running TestAddOperation() ***" << std::endl;
+  TestAddOperation();
+  std::cout << "*** Running TestComZeroExt() ***" << std::endl;
+  TestComZeroExt();
 
   return 0;
 }
```

Comment:
This can be done easier than using `std::cout` here. Simple add a JLM_REGISTER_UNIT_TEST() foreach of the individual tests and get rid of the `Test()` function. The result will be that you have 3 unit tests in this file instead of 1 unit test which invokes 3 functions (that each contain a unit test).